### ISSUE TITLE
module 'astropy.utils.data' has no attribute '_find_pkg_data_path'

### ISSUE THREAD

**Issue by moustakas** [desihub/speclite#67]
```bash
conda create -n speclite-test python=3.8 astropy=4.3
conda activate speclite-test
pip install speclite
python -c "from speclite.filters import load_filters ; load_filters('decam2014-r')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/ioannis/anaconda3/envs/speclite-test/lib/python3.8/site-packages/speclite/filters.py", line 1807, in load_filters
    filters_path = astropy.utils.data._find_pkg_data_path('data/filters/')
AttributeError: module 'astropy.utils.data' has no attribute '_find_pkg_data_path'
```

**Comment by moustakas** [comment]
Looks like this private method was renamed and made public in version 4.3 (an unreleased version)--
https://docs.astropy.org/en/latest/changelog.html#id20

I'll revert back to astropy 4.2.1 for now.

**Comment by mcbeth** [comment]
Something like this would fix it.  I'm working on permission for a PR (found and fixed at work), but it'd probably be faster for someone with commit to work on it.

```
diff --git a/speclite/filters.py b/speclite/filters.py
index 88d192a..70f5f16 100644
--- a/speclite/filters.py
+++ b/speclite/filters.py
@@ -1804,7 +1804,11 @@ def load_filters(*names):
         filters in the order they were specified.
     """
     # Replace any group wildcards with the corresponding canonical names.
-    filters_path = astropy.utils.data._find_pkg_data_path('data/filters/')
+    # astropy v4.3 replaced _find_pkg_data_path with get_pkg_data_path
+    if hasattr(astropy.utils.data,'_find_pkg_data_path'):
+        filters_path = astropy.utils.data._find_pkg_data_path('data/filters/')
+    else:
+        filters_path = astropy.utils.data.get_pkg_data_path('data/filters/')
     names_to_load = []
     for name in names:
         group_match = _group_wildcard.match(name)
```

**Comment by moustakas** [comment]
Thanks @mcbeth. For the record, this was @weaverba137's fix in `desiutil`
  https://github.com/desihub/desiutil/pull/175/commits/4ec967626d8aa26ef025f8947b94c40dc63ba01d

**Comment by mcbeth** [comment]
That also seems reasonable

**Comment by grburgess** [comment]
I opened a PR that should fix this as it doesn't rely on astropy.

**Comment by dkirkby** [comment]
Thanks for the PR @grburgess.  It looks good and I just added one minor suggestion.

**Comment by dkirkby** [comment]
Fixed with #68  - thanks @grburgess!

**Comment by rrjbca** [comment]
Is there a plan for a release including this fix any time soon? Would certainly be appreciated by downstream projects!

**Comment by dkirkby** [comment]
Released as v0.14.
